src/components/player/PlayerLeague.vue

```diff
@@ -133,6 +165,19 @@ export default class PlayerLeague extends Vue {
   get isRanked() {
     return this.modeStat.rank > 0;
   }
+
+  get lastTenMatchesPerformance(): string {
```

Comment:
Type needs to be `string[]`, otherwise it gives a type return error.

Reply:
Sure, will fix